index.bs

```diff
@@ -683,6 +683,20 @@ partial interface Gamepad {
 <dfn attribute for="Gamepad" id="gamepad-getvrdisplays-attribute">displayId</dfn>
 Return the {{VRDisplay/displayId}} for the associated {{VRDisplay}}.
 
+# Processing Model # {#proc}
+
+## submitFrame ## {#submitframe-proc}
+
+There are multiple situations where applications won't invoke {{VRDisplay}}.{{submitFrame()}} at the expected rate (the {{VRDisplay}} native framerate). The User Agent MUST intervene to protect the user from motion sickness or consistently inform her about different situations such as errors or navigation between pages. This is the breakdown of the different scenarios:
+
+* The application calls {{VRDisplay}}.{{submitFrame()}} at the expected rate, but only a few frames are dropped. The User Agent MUST re-project the last submitted frame.
+
+* The application calls {{VRDisplay}}.{{submitFrame()}} consistently below the expected framerate to a degree that could make the user feel ucnonfortable. The User agent SHOULD render a trusted interstitial that obscures or minimize the application until its performance improves. If the low framerate persists for a long time the User Agent MUST show a message indicating that the application is too slow to be presented and terminate presentation.
+
+* An application can stop calling {{VRDisplay}}.{{submitFrame()}} all toghether under several circumstances:
+    * A content exception or error has ocurred. In this case the User Agent SHOULD notify the user of the error with a trusted UI.
```

Comment:
where is the term `trusted UI` defined? the reason I ask because the [Permissions API](https://w3c.github.io/permissions/) is now the preferred mode of prompting users for (in-context) permission changes and notifications.